src/grt/src/MakeWireParasitics.cpp

```diff
@@ -93,13 +97,17 @@ void MakeWireParasitics::estimateParasitcs(odb::dbNet* net,
   }
 
   sta_net_ = network_->dbToSta(net);
-  node_id_ = 0;
   node_map_.clear();
 
-  parasitic_
-      = parasitics_->makeParasiticNetwork(sta_net_, false, analysis_point_);
-  makeRouteParasitics(net, route);
-  makeParasiticsToPins(pins);
+  for (sta::Corner* corner : *sta_->corners()) {
+    node_id_ = 0;
+    net_corner_ = corner;
+    net_analysis_point_ = net_corner_->findParasiticAnalysisPt(min_max_);
+    net_parasitic_ = parasitics_->makeParasiticNetwork(
+        sta_net_, false, net_analysis_point_);
+    makeRouteParasitics(net, route);
+    makeParasiticsToPins(pins);
```

Comment:
Is there a reason to make these class members rather than arguments to the methods?  They seem like transient data.

Reply:
They were class members before, so I left it that way to try to minimize the code changes, but it would make better sense to pass them around in the function classes (which is how the placement estimator in rsz works)

Reply:
I'll approve now and you can fix that here or in another PR

Reply:
@maliberty I went ahead and made the switch in the PR, that removed all the member variables used during the parasitic estimation